### ISSUE TITLE
probs = [F.softmax(tensr) for tensr in with_epsilon]  may be wrong?

### ISSUE THREAD

**Issue by ForgotLove** [shayneobrien/coreference-resolution#19]
The code "probs = [F.softmax(tensor) for tensor in with_epsilon] "  in class Trainer in coref.py.When i train,i get the prob(size:span_size*(antecedent_size+1)*1 ) with all cell has a fixed value 1.Maybe the right code is "probs = [F.softmax(tensor,dim=0) for tensor in with_epsilon]".
My torch version is 1.4.0.
Ps:My english is pool,i hope you can understand what i say.
